yamllint/rules/truthy.py

```diff
@@ -63,6 +71,23 @@
     yes:  1
     on:   2
     True: 3
+
+#. With ``truthy: {allowed-values: ["yes", "no"]}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    key1: yes
+    key2: no
+    key3: true
+    key4: false
```

Comment:
`true` and `false` should not be allowed with `allowed-values: ["yes", "no"]`

Reply:
Updated the documentation example.